app/models/post.rb

```diff
@@ -57,6 +57,34 @@ def address
     [street, city, state, postal_code, country].compact.join(", ")
   end
 
+  def self.filter_type(type)
+    if type == "needed" || type == "available"
+      send(type.to_sym)
```

Comment:
I was leveraging the enum helpers but that's not needed, especially when I make it all complicated by using `send` :joy: